### ISSUE TITLE
AttributeError: 'RRset' object has no attribute 'processing_order'

### ISSUE THREAD

**Issue by MRB60** [karolyi/py3-validate-email#74]
Running on localhost I see the following:
 ```
File "skip/venv/lib/python3.8/site-packages/validate_email/dns_check.py", line 40, in _get_cleaned_mx_records
    for record in answer.rrset.processing_order():  # type: MX
AttributeError: 'RRset' object has no attribute 'processing_order'
```
Is this problem really due to localhost?

**Comment by karolyi** [comment]
I would guess so. Let us know when you can provide a reproducible error.

**Comment by MRB60** [comment]
> I would guess so. Let us know when you can provide a reproducible error.

Same problem on AWS (no reverse DNS setup yet) also with deactivated SMTP check. Code:

`            is_valid = validate_email(email_address='<-removed->@gmail.com', check_format=True, check_blacklist=True, check_dns=True, dns_timeout=10, check_smtp=False, smtp_timeout=10, smtp_debug=False)`

**Comment by karolyi** [comment]
as far as I remember, we don't check for reverse DNS at all, so that shouldn't be the problem.

can you tell me the module versions you use? also the one for dnspython and idna?

**Comment by MRB60** [comment]
OK. Here are the versions used:
py3-validate-email==1.0.0
dnspython==2.0.0
idna==3.2

**Comment by karolyi** [comment]
could you update `dnspython` to latest (`2.1.0` at the time of writing this) and test again? I think there lies the culprit.

**Comment by MRB60** [comment]
Yes - worked. Thank you!

**Comment by karolyi** [comment]
you're welcome. actually, I'm trying to make a release to pypi with this fix but somehow the API is failing so I can't upload it now.

the fix will come with `1.0.1`.
